### PR TITLE
Add integrations and clean up

### PR DESCRIPTION
**Pull request by jeddy3**
> Which issue, if any, is this issue related to?

Ref https://github.com/stylelint/stylelint/pull/6573

> Is there anything in the PR that needs further explanation?

Details over in that PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,101 +1,140 @@
...
+- [Task runners](#task-runners)
+- [Testing](#testing)
 - [Tools](#tools)
 - [Tutorials](#tutorials)
 
+## Analysis platforms
+
+- [codacy-stylelint](https://github.com/codacy/codacy-stylelint) - Codacy integration.
+- [codeclimate-stylelint](https://github.com/gilbarbara/codeclimate-stylelint) - Code Climate integration.
+- [mega-linter](https://nvuillam.github.io/mega-linter) - Integrated into Mega-Linter.
```
Comment (ybiquitous) [inline README.md:26]: [suggest] The name and website has changed:

```suggestion
- [megalinter](https://megalinter.io) - Integrated into Mega-Linter.
```

File: README.md [hunk 2]
```diff
@@ -1,101 +1,140 @@
...
+- [putout](https://github.com/coderaiser/putout) - Integrated into Putout.
+- [reviewdog/action-stylelint](https://github.com/reviewdog/action-stylelint) - Reviewdog GitHub Action.
+
 ## Configs
 
-- [stylelint-recommended](https://github.com/stylelint/stylelint-config-recommended) - The recommended shareable config for Stylelint.
-- [stylelint-recommended-scss](https://github.com/stylelint-scss/stylelint-config-recommended-scss) - Same as above, when using SCSS.
```
Comment (ybiquitous) [inline README.md:19]: [ask] Are the recommended configs missing?

File: README.md [hunk 3]
```diff
@@ -1,101 +1,140 @@
...
-- [Recess Property Order](https://github.com/stormwarning/stylelint-config-recess-order) - Sorts CSS properties the way Recess did and Bootstrap does.
+- [stylelint-config-prettier-scss](https://github.com/prettier/stylelint-config-prettier-scss) - Turns off conflicting rules when using Prettier for SCSS.
+- [stylelint-config-prettier](https://github.com/prettier/stylelint-config-prettier) - Turns off conflicting rules when using Prettier for CSS.
+- [stylelint-config-primer](https://github.com/primer/stylelint-config-primer) - The GitHub shareable config.
+- [stylelint-config-rational-orderr](https://github.com/constverum/stylelint-config-rational-order) - Sorts related property declarations by grouping together in the rational order.
+- [stylelint-config-recess-order](https://github.com/stormwarning/stylelint-config-recess-order) - Sorts CSS properties the way Recess did and Bootstrap does.
+- [stylelint-config-sass-guidelines](https://github.com/bjankord/stylelint-config-sass-guidelines) - SASS guidelines config.
```
Comment (ybiquitous) [inline README.md:37]: [trivial] Typo.

```suggestion
- [stylelint-config-sass-guidelines](https://github.com/bjankord/stylelint-config-sass-guidelines) - Sass guidelines config.
```

File: README.md [hunk 4]
```diff
@@ -1,101 +1,140 @@
...
+## Custom syntaxes
+
+- [postcss-html](https://www.npmjs.com/package/postcss-html) - HTML, including Vue SFC.
+- [postcss-less](https://www.npmjs.com/package/postcss-less) - Less.
+- [postcss-linaria](https://www.npmjs.com/package/@linaria/postcss-linaria) - Linaria.
+- [postcss-lit](https://www.npmjs.com/package/postcss-lit) - Lit elements.
+- [postcss-markdown](https://www.npmjs.com/package/postcss-markdown) - Markdown.
+- [postcss-sass](https://www.npmjs.com/package/postcss-sass) - Sass.
+- [postcss-scss](https://www.npmjs.com/package/postcss-scss) - SCSS.
+- [postcss-styled-components](https://www.npmjs.com/package/postcss-styled-components) - Styled-components.
```
Comment (ybiquitous) [inline README.md:52]: [suggest] How about adding `postcss-styled-syntax` by @hudochenkov?

```suggestion
- [postcss-styled-components](https://www.npmjs.com/package/postcss-styled-components) - Styled-components.
- [postcss-styled-syntax](https://www.npmjs.com/package/postcss-styled-syntax) - CSS-in-JS.
```

File: README.md [hunk 5]
```diff
@@ -1,101 +1,140 @@
...
+- [ember-cli-stylelint](https://github.com/billybonks/ember-cli-stylelint) - Ember CLI plugin.
+- [grunt-stylelint](https://github.com/wikimedia/grunt-stylelint) - Grunt plugin.
+- [gulp-stylelint](https://github.com/olegskl/gulp-stylelint) - Gulp plugin.
+- [jest-runner-stylelint](https://github.com/keplersj/jest-runner-stylelint) - Jest plugin.
+- [nx-stylelint](https://github.com/Phillip9587/nx-stylelint) - Nx plugin.
+- [stylelint-webpack-plugin](https://github.com/webpack-contrib/stylelint-webpack-plugin) - Webpack plugin.
+
+## Testing
+
+- [stylelint-tape](https://www.npmjs.com/package/stylelint-tape) - Test custom rules using Tape.
```
Comment (ybiquitous) [inline README.md:128]: [suggest] What about adding our package `jest-preset-stylelint`?

```suggestion
- [jest-preset-stylelint](https://www.npmjs.com/package/jest-preset-stylelint) - Jest preset for Stylelint plugins.
- [stylelint-tape](https://www.npmjs.com/package/stylelint-tape) - Test custom rules using Tape.
```

File: README.md (hunk 2 above)
Comment (jeddy3) [reply README.md:19]: Yes, as they can be reached via the READMEs of the standard configs. Ironically, we don't recommend people use the recommended config. The standard configs are a better fit for the majority of users.

**Comment by jeddy3** [comment]
@ybiquitous Thanks for the review and suggestions!

**Review by ybiquitous — APPROVED** [review]
Thanks. LGTM 👍🏼

**Review by ntwb — APPROVED** [review]

**Pull request merged**